pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "netboxlabs-diode"
+version = "1.2.0"
+description = "Diode is a data ingestion service for NetBox that greatly simplifies and enhances the process of adding and updating data in NetBox"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
nit: python 3.8 reached EOL in October 2024, can we bump it to 3.10 as minimum, please?

Reply:
Done